.NET/Microsoft.Recognizers.Text.NumberWithUnit/Constants.cs

```diff
@@ -36,6 +36,11 @@ public static class Constants
         // For cases like '2:00 pm', both 'pm' and '00 pm' are not dimension
         public const string AMBIGUOUS_TIME_TERM = BaseUnits.AmbiguousTimeTerm;
 
+        // In certain cultures the unit can be split around the number,
+        // e.g. in Japanese "秒速100メートル" ('speed per second 100 meters' = 100m/s).
+        // This flag tells the recognizer to combine prefix and suffix in order to parse the unit correctly.
+        public const string PARTIAL_UNIT = "Partial unit";

```

Comment:
I'd rather have constants with no whitespace in them as much as possible. Would split_unit be a better term?

Reply:
modified name